src/promptflow/tests/executor/unittests/_core/test_tools_manager.py

```diff
@@ -197,5 +198,4 @@ def test_collect_package_tools_and_connections(self, install_custom_tool_pkg):
 
         content = templates["my_tool_package.tools.my_tool_with_custom_strong_type_connection.MyCustomConnection"]
         expected_template_str = textwrap.dedent(expected_template)
-
-        assert content in expected_template_str
```

Comment:
Why is the acutal, i.e. content has no schema link? Since expected_template has no schema link and content is in expected_template

Reply:
I think if use assert in, no need to check all string. if you want to check all string, why not use equal?